R/endpoints.R

```diff
@@ -94,27 +94,30 @@ endpoint_model_status <- function(queue) {
 
 endpoint_model_result <- function(queue) {
   function(req, res, id) {
-    response <- with_success(queue$result(id))
```

Comment:
this function has been refactored to convert it from a complicated set of nested if/elses to basically a switch over model status. All status except for `COMPLETE` will give an error response